doc/api/errors.md

```diff
@@ -2223,6 +2219,18 @@ closed.
 These errors have never been released, but had been present on master between
 releases.
 
+<a id="ERR_ENTRY_TYPE_MISMATCH"></a>
```

Comment:
Was this left in on purpose?

Reply:
Yes, see the section head:

> These errors have never been released, but had been present on master between
releases.